Prisma Cloud Compute docs update (#32943)
demisto/content: Packs/PrismaCloudCompute/README.md

This pack includes Cortex XSIAM content.

<~XSIAM>
## Overview

> This integration lets you import **Palo Alto Networks - Prisma Cloud Compute** alerts into XSIAM

Reviewer: ```suggestion
This integration lets you import **Palo Alto Networks - Prisma Cloud Compute** alerts into Cortex XSIAM.
```

## Use Cases

Manage Prisma Cloud Compute alerts in Cortex XSIAM.
You can create new playbooks, or extend the default ones, to analyze alerts, assign tasks based on your analysis, and open tickets on other platforms.

## Configure Prisma Cloud Compute

Configure Prisma Cloud Compute to send alerts to Cortex XSIAM by creating an alert profile.

> 1. Login to your Prisma Cloud Compute console, on new Prisma Cloud versions, go to Runtime Security.

Reviewer: ```suggestion
1. Log in to your Prisma Cloud Compute console. On new Prisma Cloud versions, go to **Runtime Security**.
```

1. Navigate to **Manage > Alerts**.
1. Create a new alert profile by clicking **Add Profile**.
1. Provide a name, select **Cortex** from the provider list, and select **XSOAR** under Application.
1. Select the alert triggers. Alert triggers specify which alerts are sent to Cortex XSIAM.
1. Click **Save** to save the alert profile.

## Configure Cortex XSIAM

1. Navigate to **Settings > Integrations > Instances**.
1. Search for **Prisma Cloud Compute**.
1. Click **Add instance** to create and configure a new integration.

> * **Name**: Name for the integration.
> * **Fetches incidents**: Configures this integration instance to fetch alerts from Prisma Cloud Compute.
> * **Prisma Cloud Compute Console URL**: URL address of your Prisma Cloud Compute console. Copy the address from the alert profile created in Prisma Cloud Compute, or under **Runtime Security** copy the address from **System > Utilities > Path to Console**.
> * **Prisma Cloud Compute Project Name (if applies)**: If using projects in Prisma Cloud Compute, enter the project name here. Copy the project name from the alert profile created in Prisma Cloud Compute.
> * **Trust any certificate (not secure)**: Skips verification of the CA certificate (not recommended).
> * **Use system proxy settings**: Uses the system's proxy settings.
> * **Credentials**: Prisma Cloud Compute login credentials.
> * **Prisma Cloud Compute CA Certificate**: CA Certificate used by Prisma Cloud Compute. Copy the certificate from the alert profile created in Prisma Cloud Compute.

Reviewer: ```suggestion
   * **Name**: Name for the integration.
   * **Fetches incidents**: Configures this integration instance to fetch alerts from Prisma Cloud Compute.
   * **Prisma Cloud Compute Console URL**: URL address of your Prisma Cloud Compute console. Copy the address from the alert profile created in Prisma Cloud Compute, or under **Runtime Security** copy the address from **System > Utilities > Path to Console**.
   * **Prisma Cloud Compute Project Name (if applies)**: If using projects in Prisma Cloud Compute, enter the project name here. Copy the project name from the alert profile created in Prisma Cloud Compute.
   * **Trust any certificate (not secure)**: Skips verification of the CA certificate (not recommended).
   * **Use system proxy settings**: Uses the system's proxy settings.
   * **Credentials**: Prisma Cloud Compute login credentials.
* **Prisma Cloud Compute CA Certificate**: CA Certificate used by Prisma Cloud Compute. Copy the certificate from the alert profile created in Prisma Cloud Compute.
```

4. Click **Test** to validate the integration.
5. Click **Done** to save the integration.

## Using the integration and scripts

The integration ships with four default playbooks:

> * Prisma Cloud Compute - Audit Alert v3
> * Prisma Cloud Compute - Cloud Discovery Alert
> * Prisma Cloud Compute - Compliance Alert
> * Prisma Cloud Compute - Vulnerability Alert

Reviewer: ```suggestion
* **Prisma Cloud Compute - Audit Alert v3**
* **Prisma Cloud Compute - Cloud Discovery Alert**
* **Prisma Cloud Compute - Compliance Alert**
* **Prisma Cloud Compute - Vulnerability Alert**
```

> 3 of the above playbooks (all except _Audit Alert v3_) contain a single script. The script in each playbook encode the raw JSON alerts into Cortex XSIAM objects that can then be used in the playbooks. The scripts are:

Reviewer: ```suggestion
Three of the above playbooks (all except _Audit Alert v3_) contain a single script. The script in each playbook encode the raw JSON alerts into Cortex XSIAM objects that can then be used in the playbooks. The scripts are:
```

> * PrismaCloudComputeParseComplianceAlert
> * PrismaCloudComputeParseVulnerabilityAlert
> * PrismaCloudComputeParseCloudDiscoveryAlert

Reviewer: ```suggestion
* **PrismaCloudComputeParseComplianceAlert**
* **PrismaCloudComputeParseVulnerabilityAlert**
* **PrismaCloudComputeParseCloudDiscoveryAlert**
```

To better understand how playbooks and scripts interoperate, consider the _Prisma Cloud Compute - Vulnerability Alert_ playbook.

* When the playbook is triggered, a task called **Parse Vulnerability Alert** runs.
* The task runs the **PrismaCloudComputeParseVulnerabilityAlert** script, which takes the `prismacloudcomputerawalertjson` field of the incident (the raw JSON alert data) as input.

![image](https://raw.githubusercontent.com/demisto/content/f808c78aa6c94a09450879c8702a1b7f023f1d4b/Packs/PrismaCloudCompute/doc_files/prisma_alert_raw_input.png)

> * Click **outputs** to see how the script transformed the raw JSON input into a XSIAM object.

Reviewer: ```suggestion
* Click **outputs** to see how the script transformed the raw JSON input into a Cortex XSIAM object.
```

![image](https://raw.githubusercontent.com/demisto/content/f808c78aa6c94a09450879c8702a1b7f023f1d4b/Packs/PrismaCloudCompute/doc_files/prisma_alert_outputs.png)

At this point, you can add tasks that extend the playbook to check and respond to alerts depending on the properties of the Cortex XSIAM object.

### Audit Alert v3 playbook

> This playbook is not similar to the other 3 playbooks. It is a default playbook for parsing and enrichment of Prisma Cloud Compute audit alerts.

Reviewer: ```suggestion
This playbook is not similar to the other three playbooks. It is a default playbook for parsing and enrichment of Prisma Cloud Compute audit alerts.
```

The playbook has the following sections:

Enrichment:
- Image details
- Similar container events
- Owner details
- Vulnerabilities
- Compliance details
- Forensics

> - Defender logs.

Reviewer: ```suggestion
- Defender logs
```
